wrappers/services.go

```diff
@@ -377,7 +383,7 @@ Requires={{.MountUnit}}
 Wants={{.PrerequisiteTarget}}
 After={{.MountUnit}} {{.PrerequisiteTarget}}{{range .After}} {{.}}{{end}}
 {{- if .Before}}
-Before={{ range .Before -}}{{.}} {{- end}}
+Before={{range .Before}}{{.}} {{end}}
```

Comment:
I think you'll end up with extra whitespace at the end if there's only one entry in the 'before' list.

Reply:
Yes, thanks! See https://github.com/snapcore/snapd/pull/5132#discussion_r186381427 - I will see what I can do.